models/public/colorization-v2-norebal/colorization-v2-norebal.md

```diff
@@ -0,0 +1,111 @@
+# colorization-v2-norebal
+
+## Use Case and High-Level Description
+
+The `colorization-v2-norebal` model is one of the [colorization](https://arxiv.org/pdf/1603.08511)
+group of models designed to perform image colorization. For details
+about this family of models, check out the [repository](https://github.com/richzhang/colorization).
```

Comment:
This should also explain which model from the family this is.

Reply:
I don't understand, what do you mean?

Reply:
You have two models with the same description. There should be some difference between them.

Reply:
@IRDonch added difference